jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildImageTask.java

```diff
@@ -81,10 +81,15 @@ public void buildImage()
     TaskCommon.disableHttpLogging();
 
     try {
-      AbsoluteUnixPath appRoot = TaskCommon.getAppRootChecked(jibExtension, getProject());
+      GradleRawConfiguration rawConfiguration = new GradleRawConfiguration(jibExtension);
+      boolean containerizeWar = TaskCommon.isWarContainerization(getProject(), rawConfiguration);
```

Comment:
What if we rather stored `containerizeWar` in `RawConfiguration` rather than pass it around as a separate parameter?